pkg/controller/configmap/configmap_controller.go

```diff
@@ -204,12 +211,17 @@ func (r *ReconcileConfigMap) Reconcile(request reconcile.Request) (reconcile.Res
 	if !ncpNeedChange && !agentNeedChange {
 		// Check if NCP_IMAGE changes
 		ncpImageChanged, err := r.isNcpImageChanged()
-		if err != nil {
```

Comment:
why don't we need this? perhaps you removed the wrong `err` check with line 202?

Reply:
meh I was doing too many things at the same time

Reply:
thought so